tests/tasks/labware_location/test_main.py

```diff
@@ -0,0 +1,546 @@
+import pytest
+from unittest.mock import patch, call
+from janitor.tasks.labware_location.main import sync_changes_from_labwhere
+from janitor.helpers.mysql_helpers import parse_entry
+from tests.data.test_entries import TEST_ENTRIES
+
+
+@patch("logging.info")
+def test_given_valid_db_details_when_connecting_to_db_then_check_connection_successful(
+    mock_info,
+    config,
+):
+    sync_changes_from_labwhere(config)
+    assert mock_info.has_calls(
+        call(f"Attempting to connect to {config.LABWHERE_DB['host']} on port {config.LABWHERE_DB['host']}..."),
+        call(f"MySQL connection to {config.LABWHERE_DB['db_name']} successful!"),
+        call(f"Attempting to connect to {config.MLWH_DB['host']} on port {config.MLWH_DB['host']}..."),
+        call(f"MySQL connection to {config.MLWH_DB['db_name']} successful!"),
+    )
+
+
+@pytest.mark.parametrize(
+    "mlwh_labware_locations_table",
+    [[]],
```

Comment:
Ah right, yes sorry, I didn't mention but I don't think you need to provide `parametrize` annotations if you don't want any values in the table.  You can just ask for the fixture as an argument to the test function without including this empty list of parameters for it`

Reply:
I have now removed the annotations where they are not necessary and the tests still pass.